plugins/events-manager/buddypress/profile.php

```diff
@@ -83,17 +83,36 @@
 
 	if ( user_can( $bp->displayed_user->id, 'edit_events' ) ) {
 		?>
-		<h2 class="top-padding"><?php _e( 'My posted events', 'events-manager' ); ?></h2>
+		
 		<?php
+		$myEventsTable = '<table>
+				<tr>
+					<td>
+						#_EVENTDATES<br>#_EVENTTIMES
+					</td>
+					<td>
+						#_EVENTLINK<br>#_LOCATIONNAME<br>#_LOCATIONTOWN, #_LOCATIONSTATE
+					</td>
+					<td>
+						<a href="mailto:?subject=Check out the event I\'m organizing&body=' . htmlspecialchars( "I'm organizing an Early Years Professional Development event and I thought you'd be interested. You can get the details on the EYPD site: #_EVENTURL. I hope to see you there!" ) . '">Share</a>
+					</td>
+					<td>
+						<a href="#_EDITEVENTURL"><span class="glyphicon glyphicon-edit"></span></a>
+					</td>
+				</tr>
+			</table>';
+
+
 		$args          = [
 			'owner'         => $bp->displayed_user->id,
-			'format_header' => get_option( 'dbem_bp_events_list_format_header' ),
-			'format'        => get_option( 'dbem_bp_events_list_format' ),
-			'format_footer' => get_option( 'dbem_bp_events_list_format_footer' ),
-			'owner'         => $bp->displayed_user->id,
+			'format'        => $myEventsTable,
```

Comment:
- please change `$myEventsTable` to snake_case as per https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions